src/screens/profile/components/navbar/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor } from 'src/common/enums';
+import { DEFAULT_SCREEN_PADDING } from 'src/styles';
+import { NAVBAR_BUTTON_WIDTH } from 'src/screens/profile/components/navbar/constants';
```

Comment:
```suggestion
import { NAVBAR_BUTTON_WIDTH } from './constants';
```

Reply:
Auto import, fix in [357f930](https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup-mobile/pull/9/commits/357f93093cf6dd705105610e61633e33f8b1e92b)